tests/unit/core/test_deploy_export.py

```diff
@@ -46,9 +46,11 @@
 import nemo.collections.nlp.nm.trainables.common.token_classification_nm
 from nemo import logging
 
+TRT_ONNX_DISABLED = False
```

Comment:
@ryanleary seems that you are replacing the pytest mechanism for skipping tests with your own solution relaing on a flag and then handle it manually.

May I ask: why?

The disadvantage of your solution will be a "false impression" that the tests passed, whereas relying on pytest will actually indicate that the tests were skipped...

Reply:
I don't particularly like it either, but I certainly don't like what is there now. We shouldn't skip running these export tests just because the runner doesn't have TensorRT or PyCuda installed. There should be no such requirement for exporting to TorchScript/ONNX and thus we should not skip those tests. Skipping a test does not cause the tests to error, so the fact that these tests are not running _at all_ (if you don't have TRT dependency) could easily go undetected for a long time.

Perhaps the solution should be a way to somehow break out the `TRTONNX` version of the tests, and skip those, ONLY, in the case where TRT is not installed. Do you have suggestions on how we could/should implement such a thing?

Reply:
a) the required libraries cannot be installed on machines without GPUs, e.g. my MacBook :]
b) both solutions will skip the tests once there are no required libraries installed on the system
c) once you push it to GH, our CI server has both GPUs and all required libraries installed - so will run all tests.

Do we agree on that?